### PR TITLE
HP-729 corrections after testing ( join() from html to utility )

### PR DESCRIPTION
**Pull request by mitka0260**
HP-729 Corrections after testing ( method join() went from html to utility )

### CODE REVIEW

File: angular-fe/src/app/modules/study/study-utility.ts [hunk 1]
```diff
@@ -162,6 +163,22 @@ export class StudyUtility {
...
+      fieldStudyAuthors: this.joinArrayToString(studyDetails.fieldRightColumn.entity.fieldStudy.entity.fieldAuthor),
+      fieldCustomBoolean: studyDetails.fieldCustomBoolean,
+      fieldStudyTag: studyDetails.fieldStudyTag,
+      fieldIntroduction: studyDetails.fieldIntroduction,
+      fieldContent: studyDetails.fieldContent,
+      fieldAdditionalImages: studyDetails.fieldAdditionalImages,
+      fieldStudyText: studyDetails.fieldStudyText,
+      fieldAccordion: studyDetails.fieldAccordion,
+      fieldLabel: studyDetails.fieldLabel,
+      fieldRightColumn: studyDetails.fieldRightColumn
```
Comment (madispold) [inline angular-fe/src/app/modules/study/study-utility.ts:178]: All other fields that aren't being mapped can be passed into return object with spread operator:  ...studyDetails

a.k.a

return {
  ...studyDetails,
  fieldStudyAuthors: this.joinArrayToString(studyDetails.fieldRightColumn.entity.fieldStudy.entity.fieldAuthor),
}

File: angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html [hunk 2]
```diff
@@ -7,14 +7,19 @@
             <block-content [noPaddingTop]="true">
 
                 <h1>{{study.title}}</h1>
-                <share [pageTitle]="study?.title"></share>
+
+                <div class="author">
```
Comment (madispold) [inline angular-fe/src/app/modules/study/containers/study-detail/study-detail.component.html:11]: should have ngIf check for div class="author". If author not added creates an empty div with paddings / margins

File: angular-fe/src/app/modules/study/study-utility.ts [hunk 3]
```diff
@@ -162,6 +163,22 @@ export class StudyUtility {
     };
   }
 
+  static mapStudyDetailData(studyDetails: MappedStudyPage): MappedStudyPage {
```
Comment (mitka0260) [inline angular-fe/src/app/modules/study/study-utility.ts:166]: näen, et sisse tuleb MappedStudyPage ja välja läheb ka MappedStudyPage

**Review by madispold — CHANGES_REQUESTED** [review]
Small changes

**Review by madispold — APPROVED** [review]

**Pull request merged**